forms/common-themes/src/main/resources/theme/base/admin/resources/partials/password-policy.html

```diff
@@ -30,7 +30,7 @@
             <tr ng-repeat="p in policy">
                 <td>{{p.name|capitalize}}</td>
                 <td>
-                    <input class="form-control" ng-model="p.value" ng-show="p.name != 'notUsername' "
+                    <input id="{{p.name + (p.name == 'regexPattern' && '_' + $index || '')}}" class="form-control" ng-model="p.value" ng-show="p.name != 'notUsername' "
```

Comment:
ID is only robust if you match exactly on the ID. In this case you can end up with having multiple policies with the same name, but no idea which you're actually after.